napari_aicssegmentation/widgets/workflow_thumbnails.py

```diff
@@ -0,0 +1,143 @@
+from typing import List
+
+from aicssegmentation.workflow import WorkflowDefinition
+import cv2
+import numpy as np
+from PyQt5.QtWidgets import (
+    QPushButton,
+    QHBoxLayout,
+    QVBoxLayout,
+    QLabel,
+    QWidget,
+)
+from PyQt5.QtGui import QIcon, QPixmap, QImage
+from PyQt5 import QtCore
+
+from napari_aicssegmentation.widgets.form import Form, FormRow
+from napari_aicssegmentation._style import PAGE_CONTENT_WIDTH, Style
+
+
+class WorkflowThumbnails(QWidget):
+
+    """
+    A widget containing thumbnail images for workflows.
+
+    Params:
+        workflow_defs (List[WorkflowDefinition]): List of
+            workflow definitions to display as buttons
+    """
+
+    def __init__(self, workflow_defs: List[WorkflowDefinition] = None):
+        super().__init__()
+        layout = QVBoxLayout()
+        layout.setContentsMargins(0, 0, 0, 0)
+        self.setLayout(layout)  # reset setLayout
```

Comment:
The reason I was doing this in the `load_workflows` method is because technically we're designing the widget so that we can load and reload a list of workflows - so had to reset the layout to get rid of the existing buttons. We won't really be using it that way so it's not a big deal for now I suppose.

Reply:
Moved it back into load_workflows :)